contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--2.1.0--2.2.0.sql

```diff
@@ -3230,6 +3230,25 @@ FROM sys.events e
 WHERE e.is_trigger_event = 1;
 GRANT SELECT ON sys.trigger_events TO PUBLIC;
 
+CREATE OR REPLACE VIEW sys.sysdatabases AS
```

Comment:
Please add upgrade test. Also put PR link in CR description.

Reply:
Sounds good, upgrade tests are added in the newest revision.